docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -294,6 +294,10 @@ writing data. Data types may not map the same way in both directions between
 Trino and the data source. Refer to the following sections for type mapping in
 each direction.
 
+See the `Databricks data types documentation
+<https://docs.databricks.com/spark/latest/spark-sql/language-manual/sql-ref-datatypes.html>`_
+for more information about supported types.
```

Comment:
The databaricks docs won't know which types are supported by Trino

Reply:
about available data types in Delta Lake.

Reply:
the paragraphs says "supported" and doesn't indicate it's not about types supported by something else than Trino

Reply:
Would "..about types supported by the Delta Lake connector" be specific enough?

Reply:
If I can nit-pick a little more, could be even more clear that we're talking about the Delta Lake format and not the connector:

"for more information about supported data types in the Delta Lake table format specification."

Either way, this is not the right link. I think you want: https://github.com/delta-io/delta/blob/master/PROTOCOL.md#primitive-types

The Databricks link includes all types that are in the SparkSQL language but not all of them are supported in DL

Reply:
Reworded and updated the link to point to the Delta protocol specification.